scripts/data-gen.sh

```diff
@@ -20,3 +23,7 @@ else
 			./dbgen -f -s $scale -S $chunk -C $partition -v
 		done
 fi
+
+echo "Output Directory: $DSS_PATH"
+cd $cwd
```

Comment:
Then `popd` here

Reply:
Thanks! Changed it to `pushd` and `popd`.